plugins/DataLoadRlog/CMakeLists.txt

```diff
@@ -17,25 +17,7 @@ SET(SRC
 
 add_definitions(-L /usr/local/lib)
 
-add_custom_target(
```

Comment:
do we now rely on cereal being built for this?

Reply:
This was a substitute for scons building opendbc. I thought we needed to continue linking, but it builds fine without. I can copy the command that builds cereal and opendbc to the rlog plugin as well, just in case a user comments the line in CMakeLists.txt that adds Cereal as a subdirectory if they only want the Rlog plugin